listeners/syncprogress.go

```diff
@@ -0,0 +1,82 @@
+package listeners
+
+import (
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/wallet"
+)
+
+type SyncProgress struct {
```

Comment:
Should be `SyncProgressListener`. Same for the others; these are all listeners and are used that way with the different multiwallet methods for registering listeners.